tutorindigo/plugin.py

```diff
@@ -107,8 +108,10 @@ def _override_openedx_docker_image(
             "mfe-dockerfile-post-npm-install-learning",
             """
 RUN npm install '@edx/brand@npm:@edly-io/indigo-brand-openedx@^2.0.0'{% if INDIGO_ENABLE_DARK_THEME %} --theme=dark{% endif %}
-RUN npm install '@edx/frontend-component-header@npm:@edly-io/indigo-frontend-component-header@^3.0.0'
-RUN npm install '@edx/frontend-component-footer@npm:@edly-io/indigo-frontend-component-footer@^2.0.0'
+RUN npm install '@edx/frontend-component-header@npm:@edly-io/indigo-frontend-component-header@~3.0.0'
+RUN npm install @edly-io/indigo-frontend-component-footer@^2.0.0
+
+COPY indigo/env.config.jsx /openedx/app/
```

Comment:
Are all mfe-dockerfile-post-npm-install-* patches identical? If yes, can we de-duplicate the code?

Reply:
No, there are other MFEs too like communication, ora-grading, grade-book. We don't want to install these packages for those MFEs.